spiketoolkit/study/groundtruthstudy.py

```diff
@@ -8,131 +8,145 @@
 
 from spiketoolkit.comparison.groundtruthcomparison import compare_sorter_to_ground_truth, _perf_keys
 
-from .studytools import (setup_comparison_study, run_study_sorters, get_rec_names, 
-                                        get_recordings, copy_sortings_to_npz, iter_computed_names,
-                                        iter_computed_sorting, collect_run_times)
-
-
-
+from .studytools import (setup_comparison_study, run_study_sorters, get_rec_names,
+                         get_recordings, copy_sortings_to_npz, iter_computed_names,
+                         iter_computed_sorting, collect_run_times)
 
 
 class GroundTruthStudy:
     def __init__(self, study_folder=None):
         self.study_folder = Path(study_folder)
-        
+        self._is_scanned = False
+        self.computed_names = None
+        self.rec_names = None
+        self.sorter_names = None
+
         self.scan_folder()
-        self.computed_names = []
-        
+
         self.comparisons = None
         self.exhaustive_gt = None
-    
+
     def __repr__(self):
         t = 'Groud truth study\n'
-        t += '  ' + str(self.study_folder)+ '\n'
+        t += '  ' + str(self.study_folder) + '\n'
         t += '  recordings: {} {}\n'.format(len(self.rec_names), self.rec_names)
         if len(self.sorter_names):
             t += '  sorters: {} {}\n'.format(len(self.sorter_names), self.sorter_names)
-        
+
         return t
-    
+
     def scan_folder(self):
         self.rec_names = get_rec_names(self.study_folder)
-        
         # scan computed names
-        self.computed_names = list(iter_computed_names(self.study_folder))  # list of pair (rec_name,sorter_name)
+        self.computed_names = list(iter_computed_names(self.study_folder))  # list of pair (rec_name, sorter_name)
         self.sorter_names = np.unique([e for _, e in iter_computed_names(self.study_folder)]).tolist()
+        self._is_scanned = True
```

Comment:
self.scan_folder() is in init so 
self._is_scanned is always True.
So do we need this ?

Reply:
no we don't need it anymore.

The use is:
- new study
```
study = sc.GroundTruthStudy.create(study_folder, gt_dict)
```

- old study (TODO):
```
study = sc.GroundTruthStudy.open(study_folder)
```

both class functions that create the object. Should we move the discussion to spikecomparison?